string shooter
string hitted
uint8 last_damage
uint8 hitted_health